lib/bokken/uploaders/avatar.ex

```diff
@@ -7,13 +7,28 @@ defmodule Bokken.Uploaders.Avatar do
 
   @versions [:original, :thumb]
   @extension_whitelist ~w(.jpg .jpeg .gif .png)
+  @min_file_size 10_000
+  @max_file_size 10_000_000
 
   # Whitelist file extensions:
   def validate({file, _}) do
     file.file_name
     |> Path.extname()
     |> String.downcase()
     |> then(&Enum.member?(@extension_whitelist, &1))
+
+    size = file_size(file)
+
+    cond do
+      size < @min_file_size ->
+        {:error, "File is too small. Minimum size is #{@min_file_size} bytes."}
+
+      size > @max_file_size ->
+        {:error, "File is too large. Maximum size is #{@max_file_size} bytes."}
+
+      true ->
+        {:ok, file}
+    end
   end
```

Comment:
As this is now, if the file doesn't meet the `@extension_whitelist`, but meets the file size requirements it is taken as a valid file. Remember that Elixir only returns the last evaluated expression.